integration-tests/src/test/resources/indexer/wikipedia_reindex_queries.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
I guess we don't need this new query? It's same with `wikipedia_index_queries.json` except that `rows` is missing in this query.

Reply:
The differences are:

- maxTime in the first query is different (because the second ingest pulls a subset of time)
- not asking for the now-gone `count` metric in the second query (is that what you mean by `rows`?)

Are you proposing just not doing queries at all after the reingest? It seems reasonable to validate that some data ended up in the new datasource.

Reply:
Oh I see. I missed the 1st change. It looks good to me. Thanks.